hw/top_earlgrey/data/ip/chip_spi_device_testplan.hjson

```diff
@@ -105,10 +158,16 @@
             - The testbench should check if the written and read data match.
             '''
       stage: V2
+      si_stage: SV3
+      lc_states: [ "PROD" ]
+      features: [
+        "SPI_DEVICE.MODE.TPM",
+        "SPI_DEVICE.MODE.TPM.READ_FIFO_MODE",
+      ]
       tests: ["chip_sw_spi_device_tpm"]
     }
-
     {
+      // This test is more a pinmux/padctrl test than a spi_device one. Consider reclassification.
```

Comment:
I agree. Would it also make sense (for now) to leave a "Notes for silicon targets" section in the `desc` to explain why `features` is empty?

Reply:
I think that's a good idea too. Done.